scrabble-score/example.erl

```diff
@@ -0,0 +1,25 @@
+-module(scrabble_score).
+
+-export([score/1]).
+
+score( Word ) ->
+  Scores = lists:map( fun get_char_value/1, string:to_lower( Word ) ),
+  lists:sum( Scores ).
+
```

Comment:
I'd go with a universal fold personally, which will make one pass over the input rather than 2. Something like should work.
```erlang
lists:foldl(fun(X, Sum) -> get_char_value(X) + Sum end, 0, string:to_lower( Word ))
```